WebKitBrowser/Extension/MixedContentWhiteListedOriginDomainsList.cpp

```diff
@@ -0,0 +1,71 @@
+#include "MixedContentWhiteListedOriginDomainsList.h"
+
+using std::unique_ptr;
+using std::vector;
+
+namespace WPEFramework {
+namespace WebKit {
+
+    /* static */unique_ptr<MixedContentWhiteListedOriginDomainsList> MixedContentWhiteListedOriginDomainsList::Parse(const char* jsonString)
+    {
+        // Origin/Domain pair stored in JSON string.
+        class JSONEntry : public Core::JSON::Container {
```

Comment:
implementation very similar to that from WhiteListedOriginDomainsList.cpp. Maybe some class hierarchy for that?